### PR TITLE
Overview tab(Analytics)  v2

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
## Proposed changes
 - Analytics->Overview Tab 2.0
 - Header dropdown Improvements
 - Sidebar Updated
 - Loader component improved
 
Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx [hunk 1]
```diff
@@ -86,20 +86,26 @@ const ProfileDropdown: React.FC = () => {
...
+                  }}
+                  onClick={() => tabs.changeSettingsTabs(0)}
+                >
+                  <Typography title="Go to settings">
+                    {t('header.profileDropdown.emailSet')}
+                  </Typography>
+                </Link>
+              ) : (
+                <Typography className={classes.projectRoleHint}>
+                  Please switch to a project you own to access settings
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx:106]: translation

File: litmus-portal/frontend/src/config/index.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ if (loc.protocol === 'https:') {
 } else {
   sockURL = 'ws:';
 }
+
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/config/index.ts:10]: is this change needed?

File: litmus-portal/frontend/src/views/Analytics/Overview/WorkflowDashboardCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                title={data.phase}
+              />
+              <div>
+                <Typography
+                  className={`${classes.testName} ${classes.noWrapProvider}`}
+                >
+                  {data.workflow_name}
+                </Typography>
+                <Typography className={classes.hint}>
+                  Agent: {data.cluster_name}
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/Overview/WorkflowDashboardCard/index.tsx:60]: translation

File: litmus-portal/frontend/src/views/Analytics/Overview/WorkflowDashboardCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                onClick={() => {
+                  history.push({
+                    pathname: `/workflows/analytics/${data.workflow_id}`,
+                    search: `?projectID=${projectID}&projectRole=${projectRole}`,
+                  });
+                }}
+              >
+                <AnalyticsIcon />
+              </IconButton>
+              <Typography>See Analytics</Typography>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/Overview/WorkflowDashboardCard/index.tsx:80]: translation

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@vanshBhatia-A4k9 PTAL

File: litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx:106]: Translations for analytics will go as a separate as discussed.

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx:106]: This is not for analytics, it's a header component file @Saranya-jena

**Review by arkajyotiMukherjee — APPROVED** [review]
clean coding 👍🏼 ...as discussed please do add translations in another PR @vanshBhatia-A4k9

**Review by ishangupta-ds — APPROVED** [review]

**Review by whitetiger1399 — APPROVED** [review]
Thanks for the refactoring!! Awesome

File: litmus-portal/frontend/src/config/index.ts (hunk 2 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/config/index.ts:10]: Missed this one, thanks :+1:

**Review by Saranya-jena — APPROVED** [review]
LGTM, thanks! 🚀
